Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -41,6 +50,17 @@ class ConversationSearchViewController: UITableViewController {
         tableView.register(ConversationSearchResultCell.self, forCellReuseIdentifier: ConversationSearchResultCell.reuseIdentifier)
         tableView.register(MessageSearchResultCell.self, forCellReuseIdentifier: MessageSearchResultCell.reuseIdentifier)
         tableView.register(ContactSearchResultCell.self, forCellReuseIdentifier: ContactSearchResultCell.reuseIdentifier)
+
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(yapDatabaseModified),
+                                               name: NSNotification.Name.YapDatabaseModified,
+                                               object: OWSPrimaryStorage.shared().dbNotificationObject)
+    }
+
+    @objc internal func yapDatabaseModified(notification: NSNotification) {
+        SwiftAssertIsOnMainThread(#function)
+
+        refreshSearchResults(delay: 1.0)
```

Comment:
Notice that we use a longer delay for these events, since they can be frequent.

Reply:
This could interfere with snappiness from the user input. e.g. if a dbModified happens just before the user enters a character they won't see results updated for 1s, instead of 0.25 seconds. 

We could make it more complicated to address this, but I don't see much benefit from observing dbModified notifications in this context.

Reply:
Okay, removing.

Reply:
Actually, I'm going to make the case for leaving this in - let's discuss at iOS standup.

Reply:
Okay, I've re-enable this but both "user changes" and "db modifications" have a much shorter delay and we skip the delay if we're not yet showing any results.